mcr.cpp

```diff
@@ -23,19 +26,22 @@ bool isWin(char game[3][3]){
  
 int main(){ 
     int i, j; 
-    char game[3][3] = {' '}; // Tic-tac-toe 
+    int n;
+    char game[3][3] = {{' ', ' ', ' '}, {' ', ' ', ' '}, {' ', ' ', ' '}}; // Tic-tac-toe
     char player1 = 'X'; 
     char player2 = 'O'; 
     bool turn = true; // false for player 1's turn, true for player 2's turn. Player 1 first. 
     cout << "X = Player 1" << endl << "O = Player 2" << endl; 
-    for (int n=0; n<9; n++){ 
+    for (n=0; n<9; n++){ 
         turn = !turn;  // use the not-operator to change true to false or false to true. 
         if (turn == false) 
             cout << "Player 1: "; 
         else 
             cout << "Player 2: "; 
-        cout << "Which cell to mark? i:[1..3], j:[1..3]: ";  
-        cin >> i >> j; 
+        do {
+                cout << "Which cell to mark? i:[0..2], j:[0..2]: ";
+                cin >> i >> j;
```

Comment:
Here we should determine whether the values of i and j are valid (whether they are between 0 and 2)